src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.view
+
+object OutputView {
+    private const val START_MESSAGE = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)"
+    private const val ATTEMPT_MESSAGE = "시도할 횟수는 몇 회인가요?"
+    private const val RESULT_MESSAGE = "실행 결과"
+    private const val WINNER_MESSAGE = "최종 우승자 : "
+    private const val COLON = " : "
+    private const val POSITION_SIGN = "-"
+    private const val SEPARATOR = ", "
+
+    fun printStartMessage() {
+        println(START_MESSAGE)
+    }
```

Comment:
이미 알고계실거 같지만, 이렇게 단일 표현식으로 사용할수도 있슴다!😀
```suggestion
    fun printStartMessage() = println(START_MESSAGE)
```

Reply:
이런 식의 코드를 작성할 때마다 헷갈리는게, 어느 방법이 옳은 방법일까요?
항상 고민됩니다..